unittests/CppInterOp/InterpreterTest.cpp

```diff
@@ -145,6 +151,9 @@ TEST(InterpreterTest, DISABLED_DetectResourceDir) {
 }
 
 TEST(InterpreterTest, DetectSystemCompilerIncludePaths) {
+#ifdef EMSCRIPTEN
+  GTEST_SKIP() << "Test fails for Emscipten builds";
+#endif
```

Comment:
Hmm, this is a weird spot actually. We don't expect `DetectSystemCompilerIncludePaths` to really work when we work in the browser correct ? Cause CppInterOp's `DetectSystemCompilerIncludePaths` uses `exec`

https://github.com/compiler-research/CppInterOp/blob/7caf10b1ccae74e62600b1b5fe05dc4e71db93bb/lib/Interpreter/CppInterOp.cpp#L2947-L2954

and `exec` has been marked as "won't-fix" by emscripten. So technically this is not expected to work (unless some change I guess is introduced in DetectSystemCompilerIncludePaths)

So we can prove it should be empty for this use case.

![image](https://github.com/user-attachments/assets/b014accd-bc67-40de-a77c-f90e956e8429)